cli-tests/test/cli-test.bats

```diff
@@ -21,6 +21,22 @@ setup() {
   assert_output --partial 'usage: droid [options]'
 }
 
+@test "clean removes folders from droid home" {
```

Comment:
Do you think that it's worth asserting, before the `--clean`, that these folders exist?

Reply:
I don't think it matters. It makes the test quite a bit longer and all we really care about is whether they're there afterwards or not.